src/devices/Media/VideoDevice/README.md

```diff
@@ -0,0 +1,45 @@
+# VideoDevice
+
+## Getting Started
+Before that, please make sure you install `v4l-utils`.
```

Comment:
remove non-actionable parts like `Before that, please make sure you` so basically just:

- Install `v4l-utils`

```
sudo apt-get update
sudo apt-get install v4l-utils
```